configuration/kiambu_pipeline_config.json

```
@@ -15,7 +15,8 @@
         "gpsdd_kiambu_s01e05_activation",
         "gpsdd_kiambu_s01e06_activation",
         "gpsdd_kiambu_s01e07_activation",
-        "gpsdd_kiambu_s01e08_activation"
+        "gpsdd_kiambu_s01e08_activation",
+        "gpsdd_kiambu_s01e05_Icraf_Kenyanpool_sms_ad"
```

```suggestion
        "gpsdd_kiambu_s01e05_icraf_kenya_pool_sms_ad"
```
unless this rename is going to cause problems in TextIt

---

yup, not quite sure how textit will handle this. will keep as is